src/olympia/promoted/tests/test_utils.py

```diff
@@ -29,8 +35,10 @@ def test_retrieve_stripe_checkout_session():
         stripe_retrieve.assert_called_once_with(stripe_session_id)
 
 
-@override_settings(STRIPE_API_SPONSORED_PRICE_ID="sponsored-price-id")
-def test_create_stripe_checkout_session():
+@override_settings(
+    STRIPE_API_SPONSORED_YEARLY_PRICE_ID="yearly-sponsored-price-id"
+)
+def test_create_stripe_checkout_sessio_for_verified():
```

Comment:
I believe this test is for sponsored, not verified. Also, you're missing an `n` in the word `sessio`.